build/ci/evergreen.yml

```diff
@@ -2064,24 +2150,42 @@ buildvariants:
       - name: pkg_test_docker_ubuntu2004
       - name: pkg_test_docker_debian9
       - name: pkg_test_docker_debian10
-  - name: release_msi
-    display_name: "Release (go-msi)"
+  - name: release_mongocli_msi
+    display_name: "Release MongoCLI (go-msi)"
     run_on:
       - windows-vsCurrent-small
     expansions:
       go_root: "c:\\golang\\go1.17"
       go_bin: "c:\\golang\\go1.17/bin"
     tasks:
-      - name: release_msi
+      - name: release_mongocli_msi
   - name: release_github
-    display_name: "Release (goreleaser)"
+    display_name: "Release MongoCLI (goreleaser)"
+    run_on:
+      - macos-1014-codesign
+    expansions:
+      go_root: "/opt/golang/go1.17"
+      go_bin: "/opt/golang/go1.17/bin"
+    tasks:
+      - name: release_mongocli
+  - name: atlascli_release_github
+    display_name: "Release AtlasCLI (goreleaser)"
     run_on:
       - macos-1014-codesign
     expansions:
       go_root: "/opt/golang/go1.17"
       go_bin: "/opt/golang/go1.17/bin"
     tasks:
-      - name: release
+      - name: release_atlascli
+  - name: atlascli_release_msi
```

Comment:
I will rename this variant to `release_atlascli_msi` once we are ready to release atlascli